### ISSUE TITLE
Pushed #pragma: No complaints if pops are missing

### ISSUE THREAD

**Issue by SvenMichaelKlose** [cc65/cc65#2483]
One can for example do a

`#pragma code-name (push, "CODE_INIT")`

and forget about doing a

`#pragma code-name (pop)`

with no warning by the preprocessor. That's meant to cause trouble.

**Comment by mrdudz** [comment]
Yes, sounds useful to have a warning when this happens. The devil is in the details though...
